xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 13.3.1
+JSONRPC_VERSION 13.4.0
```

Comment:
Just merged another PR, can you bump this to 13.5.0 and we'll merge this next

Reply:
thanks, bumped